packages/wonder-stuff-server-google/src/__tests__/get-delta.test.js

```diff
@@ -0,0 +1,28 @@
+// @flow
```

Comment:
Copied from render-gateway (likely to move to Wonder Stuff Server in the next PR).